sdk/elixir/README.md

```diff
@@ -60,6 +60,10 @@ The SDK support the Dagger Function by initiate it with:
 $ dagger init --sdk=github.com/dagger/dagger/sdk/elixir/runtime <name>
 ```
 
+**CAUTIONS**: Please note that `dagger` version 0.11.6 and earlier are not
+compatible with the runtime on `main` branch, please pin the sdk to `github.com/dagger/dagger/sdk/elixir/runtime@sdk/elixir/v0.11.6`
```

Comment:
```suggestion
compatible with the runtime on `main` branch. If you are using `dagger` v0.11.6, please pin the sdk to `github.com/dagger/dagger/sdk/elixir/runtime@sdk/elixir/v0.11.6`
```

nit: slight tweak to make it extra clear you only need to pin if you are on that dagger version, just IMO, non-blocking

Reply:
Thank you. 🙏